Resources/public/js/views/tabs/ez-locationviewlocationstabview.js

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-locationviewlocationstabview', function (Y) {
+    "use strict";
+    /**
+     * Provides the Location View Locations Tab view class.
+     *
+     * @module ez-locationviewlocationstabview
+     */
+    Y.namespace('eZ');
+
+    /**
+     * The Location View Locations Tab View class.
+     *
+     * @namespace eZ
+     * @class LocationViewLocationsTabView
+     * @constructor
+     * @extends eZ.LocationViewTabView
+     */
+    Y.eZ.LocationViewLocationsTabView = Y.Base.create('locationViewLocationsTabView', Y.eZ.LocationViewTabView, [Y.eZ.AsynchronousView], {
+        initializer: function () {
+            this._fireMethod = this._fireLoadLocations;
+
+            this.after('locationsChange', function (e) {
+                this.render();
+            });
+        },
+
+        render: function () {
+            var container = this.get('container'),
+                locations = null;
+
+            if (this.get('locations')) {
+                locations = [];
+                Y.Array.each(this.get('locations'), function (loc) {
+                    var locationJSON = loc.toJSON(),
+                        contentInfo = loc.get('contentInfo').toJSON();
```

Comment:
Yep, I've seen it in your PR, that's simplifies jsonification a lot ;)
I have rebased to master so now it includes your jsonifier